package/boot/uboot-envtools/files/ramips

```diff
@@ -34,7 +34,8 @@ zbtlink,zbt-wg2626)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x10000"
 	;;
 xiaomi,mir3p|\
-xiaomi,mir3g)
+xiaomi,mir3g|\
+xiaomi,rm2100)
```

Comment:
The device is called "Redmi Router AC2100", so please reflect that in the compatible, device and DTS names:
xiaomi,redmi-router-ac2100

Reply:
Sure thing, was just following what model name Xiaomi were referring to it as inside their rom.